migrations.go

```diff
@@ -57,6 +57,7 @@ func migrate_4_SwarmKey(cfg *Config) bool {
 func migrate_5_Bootstrap_node(cfg *Config) bool {
 	obns := []string{
 		"3.120.224.94",
+		"18.196.49.234",
```

Comment:
@cyberwalker82 Need to modify `migrate_7_Testnet_Bootstrap_node` as well for testnet node ip update

Reply:
You can try `13.59.69.165/tcp/43113` since the ip is not unique

Reply:
Made the changes to testnet bootstrap nodes